backend/modules/auth/provider.py

```
@@ -0,0 +1,452 @@
+    
+    def set_state(self, state: str, data: Dict[str, Any], ttl: int = 600) -> bool:
+        """Store OAuth state with TTL (default 10 minutes)"""
+        try:
+            data['created_at'] = time.time()
+            
+            if self.cipher:
+                try:
+                    json_payload = json.dumps(data)
+                    encrypted_payload = self.cipher.encrypt(json_payload.encode())
+                    encrypted_data = base64.b64encode(encrypted_payload).decode('utf-8')
+                except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)